models/model1_part2.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # SEPARATING THE DATA CLASSES
</pre><p>nit. Give the block a heading, don't use all caps comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>